ios/Pods/MobileCenter/MobileCenter-SDK-iOS/MobileCenter.framework/Headers/MSConstants.h

```diff
@@ -0,0 +1,47 @@
+#import <Foundation/Foundation.h>
```

Comment:
Shouldn't this code all be in `node_modules`? Why do we need to commit it with the repo?